### ISSUE TITLE
investigate Yellowstone on unenhanced 6502

### ISSUE THREAD

**Issue by microbmen** [a2-4am/4cade#492]
I have an un-enhanced apple //e.  running a BMOW Yellowstone card and a FloppyEMU.  I have tried a few different versions of Total Replay including the latest 5.0.  When it boots, I get the start up screen "TOTAL REPLAY" / "128K" / v5..alpha..   and it hangs.  Never loads past that. 

Hardware
Apple \\e 
Microsoft Softcard (80col, 128k, CP/M) 
Yellowstone disk controller
Floppy EMU

I don't know what else to provide that could help identify the issue.  Other disks load without issue.  Games work well when loaded as 5.25 floppy either real disk or floppy emu.

**Comment by microbmen** [comment]
seems like its something with the Yellowstone card.

**Comment by peterferrie** [comment]
If you're willing to try dedicated builds, e-mail me and we can work it out.

**Comment by wyatt-wong** [comment]
> If you're willing to try dedicated builds, e-mail me and we can work it out.

I am sorry but when I try to build the 4cade in macOS, I found there are a lot of warnings on "parallel". As far as I am concerned, the Makefile was significantly revised and incorporate the GNU parallel software, Is it possible to suppress the warnings generate from the "parallel" software ?

Furthermore, I found that everytime after I use "git pull" to retrieve the latest 4cade source and rebuild it to binary in macOS, I found the generated .2mg will have some problems after boot up in Virtual ][. If I delete 4cade folder and all the files in macOS and then re-download the whole 4cade repository with "git clone https://github.com/a2-4am/4cade" command and rebuild into binary, the generated 4cade.2mg will be able to run in Virtual ][ without any problems. How do I resolve this ?

**Comment by peterferrie** [comment]
For the parallel warnings, have you installed Parallel?

For building, you need to "make clean" every time before "make dsk" (or just use "make all" alone).
We don't support incremental builds anymore.
No need to replace the repo every time.

**Comment by wyatt-wong** [comment]
> For the parallel warnings, have you installed Parallel?
> 
> For building, you need to "make clean" every time before "make dsk" (or just use "make all" alone). We don't support incremental builds anymore. No need to replace the repo every time.

Of course I have downloaded GNU Parallel. Otherwise how can I successful build the source into binary ? The warning is something related to suppress warnings in parallel, but not about missing the parallel executable.

for building, thanks for your information about "make clean" and I will try it next time.

**Comment by wyatt-wong** [comment]
> For the parallel warnings, have you installed Parallel?
> 
> For building, you need to "make clean" every time before "make dsk" (or just use "make all" alone). We don't support incremental builds anymore. No need to replace the repo every time.

Here is the warning messages I got from Parallel.

<img width="682" alt="Screenshot 2022-08-24 at 12 59 58 AM" src="https://user-images.githubusercontent.com/10095648/186219377-f63f5c49-a5c0-4a0e-85c7-4dd49e0926ef.png">

**Comment by mr-stivo** [comment]
I think it's time you run 'parallel --citation' to silence the citation notices.  =)

**Comment by a2-4am** [comment]
@wyatt-wong please stop hijacking unrelated issues. Open a new issue if you have a concern or bug report.

**Comment by peterferrie** [comment]
Meanwhile, this is not a TR bug.  I will mark the whatever workaround as an enhancement instead.

**Comment by mgcaret** [comment]
@microbmen have you tried removing the softcard? If that fixes it, it is likely the same issue as #483.

**Comment by peterferrie** [comment]
Entirely unrelated after all.  It's the phantom read on a 6502, when reading into $D000, because the YS SP implementation used an indexed indirect write that crossed a page boundary, and the base address happened to hit a sensitive YS soft-switch.
A firmware fix is coming.  Nothing for TR here.
